qa/tasks/mgr/dashboard_v2/test_rbd.py

```diff
@@ -40,3 +46,36 @@ def test_list(self):
         self.assertEqual(img2['obj_size'], 4194304)
         self.assertEqual(img2['features_name'],
                          'deep-flatten, exclusive-lock, fast-diff, layering, object-map')
+
+    @authenticate
+    def test_create(self):
+        data = {'pool_name': 'rbd',
+                'name': 'test_rbd',
+                'size': 10240}
+        self._post('/api/rbd', data)
+        self.assertStatus(201)
+
+        # TODO: change to GET the specific RBD instead of the list as soon as it is available?
+        ret = self._get('/api/rbd/rbd')
+        rbd_names = [rbd['name'] for rbd in ret['value'] if 'name' in rbd]
```

Comment:
why is the `if 'name' in rbd` test necessary? are you expecting to received an dictionary without the `name` key in the list of rbd images result?

Reply:
I just wanted to be sure that the key I want to access is really there. But yes, it makes sense that the test will fail if the field has been renamed for example.